### ISSUE TITLE
[Accessibility] Landing Page: Roles do not have all required ARIA attributes

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#968]
**Description**
The CAMICROSCOPE heading on the landing page does not have the `aria-level` attribute.

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/ac686e52-c6af-4ae8-9d2a-03abc1837add)
